File: dubbo-service-csharp/trunk/java-hessian-server/src/main/java/com/eqying/pf/service/provider/model/Order.java
package com.eqying.pf.service.provider.model;

import java.io.Serializable;

/**
 * TODO(这个类的作用)
 *
 * @auther: renjunjie
 * @since: 2016/12/12 11:08
 */
public class Order implements Serializable {

	private String id;
	private String orderName;
	private User user;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getOrderName() {
		return orderName;
	}

	public void setOrderName(String orderName) {
		this.orderName = orderName;
	}

	public User getUser() {
		return user;
	}

	public void setUser(User user) {
		this.user = user;
	}
}

File: dubbo-service-csharp/trunk/dotnet-hessian-client/dotnet-hessian-client/MODEL/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using System;

namespace com.eqying.pf.service.provider.model
{

    /// <summary>
    /// TODO(这个类的作用)
    /// 
    /// @auther: renjunjie
    /// @since: 2016/12/12 11:08
    /// </summary>
    [Serializable]
    public class Order
    {

        private string id;
        private string orderName;
        private User user;

        public virtual string Id
        {
            get
            {
                return id;
            }
            set
            {
                this.id = value;
            }
        }


        public virtual string OrderName
        {
            get
            {
                return orderName;
            }
            set
            {
                this.orderName = value;
            }
        }


        public virtual User User
        {
            get
            {
                return user;
            }
            set
            {
                this.user = value;
            }
        }

        public override string ToString()
        {
            return $"{nameof(id)}: {id}, {nameof(orderName)}: {orderName}, {nameof(user)}: {user}, {nameof(Id)}: {Id}, {nameof(OrderName)}: {OrderName}, {nameof(User)}: {User}";
        }
    }
}
